Assets/MixedRealityToolkit-Examples/Demos/StandardShader/Scripts/MaterialMatrix.cs

```diff
@@ -66,32 +71,48 @@ public void BuildMatrix()
             {
                 for (int j = 0; j < dimension; ++j)
                 {
-                    GameObject sphere = GameObject.CreatePrimitive(PrimitiveType.Sphere);
-                    sphere.name = "Sphere" + (i * dimension + j);
-                    sphere.transform.parent = transform;
-                    sphere.transform.localPosition = position;
-                    position.x += 1.0f + positionOffset;
+                    GameObject element = GameObject.CreatePrimitive(PrimitiveType.Sphere);
+                    element.name = "Element" + (i * dimension + j);
+                    element.transform.parent = transform;
+                    element.transform.localPosition = position;
+                    element.transform.localRotation = Quaternion.Euler(localRotation);
+                    position.x += positionOffset;
 
                     Material newMaterial = new Material(material);
                     newMaterial.SetFloat(firstPropertyId, firstProperty);
                     newMaterial.SetFloat(secondPropertyId, secondProperty);
 
-                    Renderer renderer = sphere.GetComponent<Renderer>();
+                    Renderer renderer = element.GetComponent<Renderer>();
+                    MeshFilter meshFilter = element.GetComponent<MeshFilter>();
 
-                    if (Application.isPlaying)
+                    if (Application.isPlaying == true)
```

Comment:
Sorry, being nit picky, but do we really need to make this change? It's redundant.

Reply:
I agree haha, this is our teams new coding style so I must have done it subconsciously. But, I happy to revert it.